modules/core/server/views/layout.server.view.html

```diff
@@ -17,7 +17,12 @@
     <meta name="fragment" content="!" />
     {% for cssFile in cssFiles %}
     <link rel="stylesheet" href="/{{cssFile}}?c={{appSettings.commit}}" />
-    {% endfor %} {% if env === 'development' %} {# in dev load js in head, as it
+    {% endfor %} {# Sentry DSN - make sure this is above scripts #} {% if
+    sentry.enabled === true %}
+    <script>
```

Comment:
This will need `nonce="{{ nonce }}"` to work past our security stuff. See other `<script>` tags in this file.

See this in console without it:
![image](https://user-images.githubusercontent.com/87168/75099071-11e00100-55c6-11ea-87c9-9cbbb8e7dafa.png)

Reply:
Ah, yes! I did the manually on dev2.trustroots.org after noticing that, but didn't update it properly...

Reply:
Fixed in https://github.com/Trustroots/trustroots/pull/1264/commits/81e8ff1c47eb2854468fc3d52b27a8d1b41b139d